tests/unit/v1/test_base_query.py

```diff
@@ -252,7 +247,7 @@ def _where_unary_helper(self, value, op_enum, op_string="=="):
 
         field_pb = new_query._field_filters[0]
         expected_pb = StructuredQuery.UnaryFilter(
-            field=StructuredQuery.FieldReference(field_path=field_path), op=op_enum,
+            field=StructuredQuery.FieldReference(field_path=field_path), op=op_enum
```

Comment:
Not sure if this change is necessary, but at the same time I don't believe its problematic.